rfd/0086/config.md

```diff
@@ -0,0 +1,180 @@
+## Configuration syntax improvements
+
+JSON as a configuration language leaves much to be desired. It has no comments, is unforgiving in editing (we've specifically written code in ContainerPilot to point out extraneous trailing commas in the config!). Any configuration language change should also support those users we know who are generating ContainerPilot configurations automatically.
+
+We will abandon JSON in favor of the more human-friendly YAML configuration language. It has a particular advantage for those users who are generating configuration because of the ubiquity of YAML-generating libraries (this is its major advantage over Hashicorp HCL). During ContainerPilot configuration loading, we can check for files in `/etc/containerpilot.d/` (a configurable location via the `-config` flag) and merge them together.
```

Comment:
We shouldn't remove the config location option entirely because we don't know anything about the ultimate location we're being deployed. Yes it's usually going to be a Docker container, but... 

That being said I'm game for removing `file://` -- we've never used the "pass the config in via environment" except in some failed experiments.  Making `CONTAINERPILOT` have a sane default sounds good too. We also already have a `-config` flag.